### ISSUE TITLE
Conversion reports Oct 2020-10-19; 2020-10-20

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/data#189]
Candidate for Oct edition.
http://98.212.84.66:2200/col_plus/

There is a new taxonomic checklist:
Alucitoidea

7 databases have been updated:
ITIS
Pterophorodea
Scarabs
Scorpion Files
ThripsWiki
World Ferns
World Plants

**Comment by yroskov** [comment]
**WPlants**

- [x] Fixed
Order Not assigned • 91 living spp • WCSP
Family Sarcolaenaceae • 71 of 68 est. living spp (100%) • WCSP
Family Sphaerosepalaceae • 20 of 20 est. living spp (100%) • WCSP

**Comment by yroskov** [comment]
- [ ] truncated authorstring in names with **"comb. ined."** comment:

Should be (ac19): Phlegmariurus apolinari-mariae (Nessel) comb. ined. (provisionally accepted name) 
http://www.catalogueoflife.org/annual-checklist/2019/details/species/id/b3bcbd27735db81c505fcf2a9f49d41d

Now displayed: Phlegmariurus apolinari-mariae  (Nessel) (provisionally accepted name)
http://dev4.species.id:2200/col_plus/details/species/id/79767be9a1bc5edb525a59fd385223bd

Correct in Clearinghouse: https://data.catalogue.life/catalogue/3/assembly?assemblyTaxonKey=307297d5-54f5-43b9-abd0-3d3c845cd850

Incorrect in preview: https://preview.catalogue.life/data/taxon/23f57f50-4df7-440f-bcad-ca646f89a3a1
![image](https://user-images.githubusercontent.com/28574252/96897078-ddbe2800-1453-11eb-9ac4-69f2e6496e3d.png)

@mdoering Bug somewhere

**Comment by mdoering** [comment]
Good catch. The source looks fine: https://data.catalogue.life/catalogue/3/dataset/1140/taxon/Lycopodiales-Lycopodiaceae-Huperzioideae-Phlegmariurus-apolinari-mariae-(Nessel)

The `comb. ined.` part of the authorship is given as the name usages namePhrase in the verbatim data. And the name correct has Name.status=manuscript:

https://data.catalogue.life/catalogue/3/dataset/1140/verbatim?type=col:Name&col:ID=Lycopodiales-Lycopodiaceae-Huperzioideae-Phlegmariurus-apolinari-mariae-(Nessel)&termOp=OR

https://data.catalogue.life/catalogue/3/dataset/1140/verbatim?type=col:Taxon&type=col:Synonym&type=col:NameRel&type=col:TypeMaterial&col:nameID=Lycopodiales-Lycopodiaceae-Huperzioideae-Phlegmariurus-apolinari-mariae-(Nessel)&col:nameID=Lycopodiales-Lycopodiaceae-Huperzioideae-Phlegmariurus-apolinari-mariae-(Nessel)&col:nameID=Lycopodiales-Lycopodiaceae-Huperzioideae-Phlegmariurus-apolinari-mariae-(Nessel)&col:relatedNameID=Lycopodiales-Lycopodiaceae-Huperzioideae-Phlegmariurus-apolinari-mariae-(Nessel)&col:nameID=Lycopodiales-Lycopodiaceae-Huperzioideae-Phlegmariurus-apolinari-mariae-(Nessel)&termOp=OR

Last import was 30th Sept.
Last sync 2nd Oct.

Maybe the sync did miss sth...

**Comment by yroskov** [comment]
- [ ] Something very much wrong with estimate figures in monthly edition. They did not match figures in AC19. But we did not update any such figures since ac19. @gdower What we can do?
Example: 
ac19: Phylum Tracheophyta • 345,676 of **348,000** est. living spp
now: Phylum Tracheophyta • 348,689 of **340,000** est. living spp

(Michael Hassler pointed this with critics).

**Comment by gdower** [comment]
@yroskov, the estimation source for Tracheophyta is different:

ac19: Roskov Y. after Govaerts R. (2019). Personal comment.
now: Roskov Y. after Govaerts R. (2015). Personal comment.

**Comment by yroskov** [comment]
**2020-10-28** at http://dev4.species.id:9191/col_plus/

- [x] Spp counts matched expectation. ITIS Aug/Sep 146683 vs 146991.

**Comment by yroskov** [comment]
**2020-10-28**:
@gdower 

- [ ] Incorrect credits in the classification on Species page (at least, in WFerns & Plants):

![image](https://user-images.githubusercontent.com/28574252/97597653-14f08400-19d4-11eb-8dc9-1c356ecf4d15.png)

**Comment by yroskov** [comment]
- [ ] There is no "how to cite the source" (bibliographic citation) on Species page.

**Comment by yroskov** [comment]
Database Qualifiers on Species page:
![image](https://user-images.githubusercontent.com/28574252/97599533-1ae76480-19d6-11eb-985c-fc260c52fb72.png)

- [ ] Geographic Coverage is missing among Qualifiers.
- [ ] Unclear what means % and stars (context message is needed).
